fedora_manual_setup.md

```diff
@@ -0,0 +1,114 @@
+# Manual setup on Fedora (34)
+
+## Language and runtime version management
+
+In order to manage the required languages and runtime: Ruby, Python and Node.js, we recommend using [asdf](https://asdf-vm.com/guide/introduction.html). See its [getting started](https://asdf-vm.com/guide/getting-started.html) guide in order to get specific instructions for your SHELL and installation method.
+
+Once installed, use the provided `.tool-versions.sample` file to get the appropriate versions.
+```
+ln -s .tool-versions.sample .tool-versions
+```
+
+## Ruby and Node.js
+
+The project supports **[ruby 2.6.x](https://www.ruby-lang.org/en/downloads/)** and **[Node.js 12](https://nodejs.org/en/download/)**.
+The recommended way to install them is with `asdf`:
+
+```
+asdf plugin add ruby
+asdf plugin add nodejs
+
+asdf install
+```
+
+> Alternatively, Node.js can be installed as a [Module](https://developer.fedoraproject.org/tech/languages/nodejs/nodejs.html):
+> ```
+> dnf module install nodejs:12
+> ```
+
+## Dependencies
+
+```
+sudo yum install patch autoconf automake bison libffi-devel libtool libyaml-devel readline-devel sqlite-devel zlib-devel openssl-devel ImageMagick ImageMagick-devel mysql-devel postgresql-devel chromedriver memcached
+
+sudo systemctl restart memcached
```

Comment:
Not sure about installing memcached, b/c I use it in a container, and the APP will work even without it anyway

Reply:
I don't use it either. Actually, there is no step or instruction to start it up as part of macOS setup.